src/games/strategy/triplea/delegate/BattleCalculator.java

```diff
@@ -944,171 +933,6 @@ public static CasualtyDetails selectCasualties(final String step, final PlayerID
     return sortedWellEnoughUnitsList;
   }
 
-  private static List<Unit> sortUnitsForCasualtiesWithSupportNew(final Collection<Unit> targetsToPickFrom,
```

Comment:
This can be removed. I created the new version of this that uses the caching to speed things up so this isn't used anymore.